javascript/javascript1/week1/script.js

```diff
@@ -25,3 +22,61 @@ if (shouldShowResultInDogYears === true) {
     `Your dog will be ${humanYear} human years old in ${dogYearFuture}`
   );
 }
+
+//Housey pricey (A house price estimator)
+let wide = null;
+let deep = null;
+let high = null;
+let gardenSizeInM2 = null;
+
+function volumeInMeters(wide, deep, high) {
+  let volumeInMeters = wide * deep * high;
+  return volumeInMeters;
+}
+
+function housePrice(volumeInMeters, gardenSizeInM2) {
+  let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+  return housePrice;
+}
+
```

Comment:
very nice with using functions. In both of them I suggest using `const` instead of `let`.